### PR TITLE
[Android/tflite] Let NDK know TFLITE_VERSION @open sesame 10/12 10:32

### PR DESCRIPTION
**Pull request by anyj0527**
- After support for tf2-lite, TFLITE_VERSION is required. Let NDK know it
- For Android, the name of subplugin should be always "tensorflow-lite"
  not "tensorflow2-lite"
- Add NNS_API_FLAGS to compile flags for sub-plugins

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc [hunk 1]
```diff
@@ -1078,8 +1078,10 @@ tflite_checkAvailability (accl_hw hw)
 
 #if TFLITE_VERSION == 1
 static gchar filter_subplugin_tensorflow_lite[] = "tensorflow-lite";
-#else
+#elif TFLITE_VERSION == 2
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc:1081]:
```
#if TFLITE_VERSION == 2
static gchar filter_subplugin_tensorflow_lite[] = "tensorflow2-lite";
#else
static gchar filter_subplugin_tensorflow_lite[] = "tensorflow-lite";
```

Comment (anyj0527) [reply ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc]: Updated!

**Comment by anyj0527** [comment]
Updated: Fix `api/android/build-android-lib.sh` to deal with `tf_lite_ver="2.3.0"`

**Comment by jaeyun-jung** [comment]
Discussion @anyj0527 @dongju-chae 
#2760 and this PR about tf-lite version

**Comment by anyj0527** [comment]
I'll update this PR after #2760

File: api/android/api/src/main/jni/Android.mk [hunk 2]
```diff
@@ -62,9 +68,20 @@ endif
...
-NNS_API_FLAGS += -DENABLE_TENSORFLOW_LITE=1
+NNS_API_FLAGS += \
+    -DENABLE_TENSORFLOW_LITE=1 \
+    -DTFLITE_VERSION=$(TFLITE_VERSION) \
+    -DTFLITE_VERSION_MAJOR=$(TFLITE_VERSION_MAJOR) \
+    -DTFLITE_VERSION_MINOR=$(TFLITE_VERSION_MINOR) \
+    -DTFLITE_VERSION_MICRO=$(TFLITE_VERSION_MICRO)
+
 # define types in tensorflow-lite sub-plugin. This assumes tensorflow-lite >= 1.13 (older versions don't have INT8/INT16)
 NNS_API_FLAGS += -DTFLITE_INT8=1 -DTFLITE_INT16=1
```
Comment (jaeyun-jung) [inline api/android/api/src/main/jni/Android.mk:79]: let's consider tf-lite ver here (tf-lite > 1.13)

Comment (anyj0527) [reply api/android/api/src/main/jni/Android.mk]: Updated! And move this part to `Android-tensorflow-lite-.mk`

**Review by jaeyun-jung — APPROVED** [review]

**Review by myungjoo — APPROVED** [review]

**Pull request merged**